src/main/java/com/project/Investment/App/config/AppBeanConfig.java

```diff
@@ -19,4 +23,24 @@ public JdbcTemplate jdbcTemplate(DataSource dataSource) {
     public Runner runner() {
         return new Runner();
     }
+
+    @Bean
+    public Connection connection() {
+        String URL = "jdbc:h2:mem:investment";
```

Comment:
Do not store your passwords in git. Never! 
You should store your creds in environment variables and access them via application.yml
Refactor this config to use `Properties`

Reply:
Done